manifest.json

```diff
@@ -27,7 +27,7 @@
     "vtex.search-result": "2.x",
     "vtex.category-menu": "1.x",
     "vtex.breadcrumb": "0.x",
-    "vtex.login": "1.x",
+    "vtex.login": "2.x",
```

Comment:
@brunojdo I had to update the login version  to use the dreamstore iconPack

Reply:
He said not to change it. Revert it... Will not work for now, but that's ok

Reply:
ok. doing it right now

Reply:
done.